### DISCUSSION TITLE
Multiple deletion after multiple selection

### DISCUSSION THREAD

**Discussion by mincheol-jeong** [Budibase/budibase#9254] [Help]
I am trying to delete multiple by selecting duplicates. How should I implement it?
![스크린샷 2023-01-06 오후 2 08 00](https://user-images.githubusercontent.com/78635957/210934333-4f321981-09f4-422a-874a-a6243ace37cc.png)

**Comment by melohagan** [comment]
Hey @mincheol-jeong 

There is an open bug for handling deleting multiple selected rows. At the moment you can only delete one row at a time: https://github.com/Budibase/budibase/issues/8715
